### PR TITLE
Update translation proposals from Google Translate

### PR DESCRIPTION
**Pull request by bernard-lin**
**What**: Updated translations from Google Translate

**Why**: Fixed some grammar and style issues

**How**: Used https://discountry.github.io/react/ for guidance

### CODE REVIEW

**Comment by kentcdodds** [comment]
Sweet! Could @liadbiz review this please?

File: pages/index/content/zh/index.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: glamorous - 反应组件样式解决💄
-tagline: 可维护的CSS与反应
+title: glamorous - React组件style解决方案💄
```
Comment (liadbiz) [inline pages/index/content/zh/index.md]: React组件样式解决方案

Comment (bernard-lin) [inline pages/index/content/zh/index.md]: looks good to me!

**Comment by kentcdodds** [comment]
Awesome! Thanks friends!

**Comment by bernard-lin** [comment]
Now the fun begins! 😄

**Comment by liadbiz** [comment]
@kentcdodds .....wait...

**Comment by kentcdodds** [comment]
Haha, whoops! Feel free to open another pull request 😀 sorry about that

**Comment by bernard-lin** [comment]
@liadbiz does another change need to be made?

**Comment by liadbiz** [comment]
@bernard-lin yeah, but I am not sure.....
@kentcdodds sorry, I type Englinsh not that fast...
How about talking at Gitter first, and let @bernard-lin  make another PR?

**Comment by bernard-lin** [comment]
Sure! No worries, you can submit it too!

**Pull request merged**